axelrod/interaction_utils.py

```diff
@@ -87,6 +88,32 @@ def compute_normalised_cooperation(interactions):
     return normalised_cooperation
 
 
+def compute_state_distribution(interactions):
+    """
+    Returns the count of each state for a set of interactions.
+    """
+    if len(interactions) == 0:
```

Comment:
I just copied what was done in some of the other functions...should I change them all?